source/leuven/wice_quick_start.rst

```diff
@@ -50,109 +50,108 @@ more information on the following pages:
-
-.. note::
+   It is also possible to submit interactive jobs to the other partitions
+   (e.g. ``batch``, ``gpu`` or ``bigmem``) in case you need more resources.
+   For large amounts of compute resources, however, we recommend to use
+   batch jobs since these will result in fewer idling resources
+   compared to interactive jobs.
 
-   We urge our users to recompile their software on wICE.
-   Using a software on wICE which is formerly compiled on Genius can lead to confusing
-   error messages, specifically in case of MPI applications.
-   This is true even for (mini)conda environments.
-   So, as a general rule of thumb, we propose that your software (including conda) be 
-   recompiled on wICE and put in a separate folder to distinguish the build from existing
-   software that suit Genius.
 
 .. _submit to wice big memory node:
 
-Submit to a big memory node
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Submit to nodes with more memory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+IceLake nodes with 2 TiB of RAM are placed in the ``bigmem`` partition.
+To select the maximum amount of memory per core (28 000 MB, which is also the
+default), you can submit a job as follows::
 
-The big memory nodes (2048GB of RAM) are also located in the ``bigmem`` partition. 
-In case of the big memory nodes it is also important to add your memory requirements 
-(the maximum of memory per core that can be requested is 28000MB/core), for example::
+   $ sbatch --account=lp_myproject --clusters=wice --partition=bigmem \
+            --nodes=2 --ntasks-per-node=72 --mem-per-cpu=28000M myjobscript.slurm
 
-   $ sbatch --account=lp_myproject --clusters=wice --time=01:00:00 --nodes=2 \
-            --ntasks-per-node=72 --partition=bigmem --mem-per-cpu=28000M myjobscript.slurm
+There is also one IceLake node with even more memory (8 TiB RAM) in the
+``hugemem`` partition (defaulting to ``--mem-per-cpu=111900M``).
+In contrast to :ref:`Superdome <superdome_quick_start>`, you do not need to
+request entire sockets and so the node is more similar to the other large
+memory nodes in this regard.
 
 
 .. _submit to wice GPU node:
 
 Submit to a GPU node
 ~~~~~~~~~~~~~~~~~~~~
 
-The GPU nodes are located in the ``gpu`` partition, so you will need to explicitly specify 
-it when submitting your job. 
-Similar to the other nodes, the GPU nodes can be shared by different jobs from different users.
-However every user will have exclusive access to the number of GPUs requested. 
-If you want to use only 1 GPU of type A100 you can submit for example like this::
+The nodes with A100 GPUs are located in the ``gpu`` partition. As for the other
```

Comment:
will we here already mention the gpu_a100 partition?

Reply:
Those ones weren't there yet when I opened the PR, but in the meantime this has been deployed. So I've also added those partition aliases now in b13491a50fdf23618ac71ba8e7ce886d4bd45df5.